src/balancerd/src/lib.rs

```diff
@@ -201,20 +201,30 @@ impl BalancerService {
                     .unwrap_or_else(|| String::from("unknown"));
                 if let Some(provider) = cfg.cloud_provider.clone() {
                     builder.add_context(
-                        ld::ContextBuilder::new(provider)
-                            .kind("cloud_provider")
-                            .set_string("cloud_provider_region", region)
-                            .build()
-                            .map_err(|e| anyhow::anyhow!(e))?,
+                        ld::ContextBuilder::new(format!(
+                            "blncr/{}/{}/{}",
```

Comment:
I think we should emit `blncr` in the key since you're using a dedicated `balancer` context kind.

```suggestion
                            "{}/{}/{}",
```